README.md

```diff
@@ -21,6 +21,28 @@ or by setting the `FLUX_PMI_CLIENT_METHODS` environment variable globally:
 FLUX_PMI_CLIENT_METHODS="simple pmix single"
 ```
 
+### installation
+
+##### Building From Source
+
+```console
+./autogen.sh
+./configure --localstatedir=/var/
+make -j
+make check
+```
+
+To configure flux-pmix with a side-installed version of flux-core, you will
+need to make sure that side install's pkg-config files are exported. To do that,
+something like this will work:
+
+```console
+export PKG_CONFIG_PATH=${flux_install}/lib/pkgconfig:$(pkg-config --variable pc_path pkg-config)
+```
+
+Run that before `make`.
```

Comment:
Usually the PKG_CONFIG_PATH needs to be set before `./configure`?

I usually use a script similar to this
```
#!/bin/bash
PMIX_PREFIX=/opt/openpmix-4.2.2
OMPI_PREFIX=/opt/openmpi-5.0.0rc9
FLUX_CORE_PREFIX=/opt/flux-core-v0.46.1-54

PKG_CONFIG_PATH=$(pkg-config --variable pc_path pkg-config)
#PKG_CONFIG_PATH=${OMPI_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}
PKG_CONFIG_PATH=${PMIX_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}
PKG_CONFIG_PATH=${FLUX_CORE_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}

PKG_CONFIG_PATH=${PKG_CONFIG_PATH} \
     ./configure
```